src/android/HyperSDKPlugin.java

```diff
@@ -191,6 +203,45 @@ public void preFetch(JSONArray args) {
         }
     }
 
+    private void openPaymentPage(JSONArray args) {
+        synchronized (lock) {
+            try {
+                isHyperCheckoutLiteInteg = true;
+                FragmentActivity activity = (FragmentActivity) cordova.getActivity();
+                if (activity == null) {
+                    SdkTracker.trackBootLifecycle(
+                        LogSubCategory.LifeCycle.HYPER_SDK,
+                        LogLevel.ERROR,
+                        SDK_TRACKER_LABEL,
+                        "initiate",
+                        "activity is null"
+                    );
+                    return;
+                }
+                JSONObject params = new JSONObject(String.valueOf(args.get(0)));
+                HyperCheckoutLite.openPaymentPage(activity, params, new HyperPaymentsCallbackAdapter() {
```

Comment:
We need to do this in a separate activity, as is done for the process call in the plugin.

CC: @Subhash703